### PR TITLE
134 Enhance CLI output

### LINKED ISSUE #134: Enhance the CLI UX
Use the Rich package to improve the rendering of Bodywork deployment information for end users.

### PR DESCRIPTION
**Pull request by AlexIoannides**
This PR enhances the CLI by using Rich to render data and format text in the terminal. It closes #134

### CODE REVIEW

**Comment by AlexIoannides** [comment]
I've also added an alias for the `bodywork` command, so that `bw` can be used instead - e.g.,

```
$ bw service display
```

**Comment by AlexIoannides** [comment]
Example of new log formatting:

image

**Comment by AlexIoannides** [comment]
All `bodywork XX display --name` commands now work as follows:

**without `--name` specified returns a list of objects:**

image

**specifying the `--name` argument pulls-up the specific details:**

image

File: src/bodywork/cli/secrets.py [hunk 1]
```diff
@@ -133,27 +134,25 @@ def display_secrets(
...
         return None
+    secrets = k8s.list_secrets(namespace, group)
     if secret_name and not group:
-        print("please specify which secrets group the secret belongs to.")
+        print_warn("Please specify which secrets group the secret belongs to.")
         return None
+    elif secret_name and group:
+        try:
+            complete_secret_name = _create_complete_secret_name(group, secret_name)
+            print_dict(secrets[complete_secret_name].data, complete_secret_name)
```
Comment (Marios85) [inline src/bodywork/cli/secrets.py:146]: That's nice that we now have a standardised way of printing a Dictionary of values.

File: src/bodywork/cli/terminal.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    table = Table(title=f"{table_name if table_name else ''}", title_style="bold")
+    table.add_column(f"[yellow]{key_col_name}[/yellow]", style="bold purple")
+    table.add_column(f"[yellow]{val_col_name}[/yellow]", style="bold green")
+    for field, value in the_dict.items():
+        table.add_row(str(field), str(value))
+    console.print(table)
+
+
+def print_pod_logs(logs: str, stage_name: str) -> None:
+    """Render pod lods.
```
Comment (Marios85) [inline src/bodywork/cli/terminal.py:41]: Typo

**Review by Marios85 — APPROVED** [review]
This is awesome! A great improvement on what we had before!

File: src/bodywork/cli/secrets.py (hunk 1 above)
Comment (AlexIoannides) [reply src/bodywork/cli/secrets.py:146]: Yeah - makes it so much easier!

File: src/bodywork/cli/terminal.py (hunk 2 above)
Comment (AlexIoannides) [reply src/bodywork/cli/terminal.py:41]: Good spot!

**Pull request merged**